config/test.json

```diff
@@ -3,6 +3,31 @@
   "mongodb": "mongodb://localhost:27017/giveth-test",
   "givethFathersBaseUrl": "http://localhost:3040",
   "useProjectOwnerWhitelist": false,
+  "blockchain": {
+    "nodeUrl": "ws://localhost:8547",
+    "homeNodeUrl": "ws://localhost:8547",
```

Comment:
Intentionally added invalid address for local blockchain address, because if local network is up or down we get different error messages so in this way we are sure that local network is down